src/webui/www/private/confirmtrackerdeletion.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="${LANG}">
+
+<head>
+    <meta charset="UTF-8">
+    <title>QBT_TR(Remove tracker)QBT_TR[CONTEXT=confirmDeletionDlg]</title>
+    <link rel="stylesheet" href="css/style.css?v=${CACHEID}" type="text/css">
+    <script src="scripts/lib/MooTools-Core-1.6.0-compat-compressed.js"></script>
+    <script src="scripts/lib/MooTools-More-1.6.0-compat-compressed.js"></script>
+    <script>
+        "use strict";
+
+        window.addEvent("domready", () => {
```

Comment:
FYI, I'm doing some work to migrate away from mootools and starting from the easy parts. It would be great if you can follow the trend:
```suggestion
        window.addEventListener("DOMContentLoaded", () => {
```

Also please read the Notice in case you missed it: https://github.com/qbittorrent/qBittorrent/tree/master/src/webui/www#notices

Reply:
I did miss that notice - thanks for sharing it. I'll make these changes.